desktop-gnome/wike/autobuild/defines

```diff
@@ -0,0 +1,6 @@
+PKGNAME=wike
+PKGSEC=gnome
+PKGDEP="dconf glib gtk-4 hicolor-icon-theme libadwaita libsoup-3 pango python-3 pygobject-3 webkitgtk"
```

Comment:
You probably meant `webkit2gtk` here. `webkitgtk` has long since been deprecated.